docs/source/user-guide/filter-help.rst

```diff
@@ -0,0 +1,91 @@
+.. _user-guide/filter-help:
+
+Filter Help
+===========
+
+Filtering test results in Ibutsu's frontend is designed expose SQLAlchemy's filtering operations to
+the user. For all filters, the following translation is used:
+
+.. list-table:: Ibutsu Filters
+   :header-rows: 1
+
+   * - Operator
+     - String Name
+     - SQLAlchemy Example
+     - Description
+   * - =
+     - eq
+     - ``Result.<column> == <value>``
+     - Equals
+   * - !
+     - ne
+     - ``Result.<column> != <value>``
+     - Does not equal
+   * - >
+     - gt
+     - ``Result.<column> > <value>``
+     - Greater than
+   * - <
+     - lt
+     - ``Result.<column> < <value>``
+     - Less than
+   * - )
+     - gte
+     - ``Result.<column> >= <value>``
+     - Greater than or equal to
+   * - (
+     - lte
+     - ``Result.<column> <= <value>``
+     - Less than or eqaul to
+   * - ``*``
+     - in
+     - ``Result.<column>.in_(<values>)``
+     - Column ``in`` list of potential values
+   * - ~
+     - regex
+     - ``Result.<column>.op("~")(<value>)``
+     - Regex match that ``column`` contains ``value``
+   * - @
+     - exists
+     - ``Result.<column> != None`` or ``Result.<column> == None``
+     - Column exists and is defined
+
+The information for this table is taken from
+`filters.py`_.
```

Comment:
While you're fixing the issue below, just pop this on the previous line? It's so lonely here on this line by itself...

Reply:
words get lonely too sometimes